doc/source/whatsnew/v0.13.0.rst

```diff
@@ -123,21 +119,30 @@ API changes
 
   This prevents doing boolean comparison on *entire* pandas objects, which is inherently ambiguous. These all will raise a ``ValueError``.
 
-  .. code-block:: python
+  .. code-block:: text
+
+     >>> if df:
+             pass
 
-      if df:
-         ....
-      df1 and df2
-      s1 and s2
+     ValueError: The truth value of a DataFrame is ambiguous.  Use a.empty,
+     a.bool(), a.item(), a.any() or a.all().
+
+     >>> df1 and df2
+     ValueError: The truth value of a DataFrame is ambiguous.  Use a.empty,
```

Comment:
did you see if this generates a warning when building the docs? we may need to use this syntax if that's the case: https://docs.python.org/3/library/doctest.html (see `Traceback`...)

Reply:
This throws an indention error and then an undefined error when the df1 and df2 gets indented (to aling with the if df: above I think.  Since df1, df2, s1, and s2 are never defined I wasn't sure how to proceed.  Should I define them or follow the traceback approach?

Reply:
I ended up assigning the variables, let me know if you'd like to see an alternative solution...